packages/deno/rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import commonjs from '@rollup/plugin-commonjs';
+import typescript from '@rollup/plugin-typescript';
+import dts from 'rollup-plugin-dts';
+
+export default [
+  {
+    input: ['src/index.ts'],
+    output: {
+      dir: 'build',
+      sourcemap: true,
+      preserveModules: false,
+      strict: false,
+      freeze: false,
+      interop: 'auto',
+      format: 'esm',
+      banner: '/// <reference types="./index.d.ts" />',
+    },
+    plugins: [
+      nodeResolve({
+        extensions: ['.mjs', '.js', '.json', '.node', '.ts', '.tsx'],
+      }),
+      commonjs(),
+      replace({
+        preventAssignment: true,
+        values: {
+          __DEBUG_BUILD__: true,
```

Comment:
h: We want to replace `__DEBUG_BUILD__` with `(typeof __SENTRY_DEBUG__ === 'undefined' || __SENTRY_DEBUG__)`.

See https://github.com/timfish/sentry-javascript/blob/ffb3f9e06405bf46b00a238a5ee95d96a5f8d8ac/rollup/plugins/npmPlugins.js#L102